app/components/ui/Banner/Banner.jsx

```diff
@@ -4,12 +4,10 @@ import styles from './Banner.scss';
 export default function Banner() {
   return (
     <div className={styles.bannerContainer}>
-      <strong>CORONAVIRUS COVID-19: </strong>
-      Many organizations and services have reduced hours and availability. See our new
-      {' '}
-      <a className={styles.bannerLink} href="/covid">Resource Guide</a>
-      {' '}
-      - updated daily.
+      <span>
+        <strong>CORONAVIRUS UPDATE: </strong>
+        Many organizations and services have reduced hours and availability.
+      </span>
```

Comment:
Oh are we removing the link to the resource guide here?

Reply:
Yes